edgecompute/examples/traffic-routing/redirect-url (EKV)/README.md

```diff
@@ -0,0 +1,21 @@
+# url-redirects (EKV)
+
+*Keyword(s):* url-redirects, EKV<br>
+
+This example uses EdgeWorkers + EdgeKV to extract a query string product SKU from an incoming URL, match it against a SKU stored in a KV database, and properly redirect the browser to the new product page.
+
```

Comment:
Can sample data be added to this repo, along with instructions on how to populate the data into EKV?